src/gss_ntlmssp.h

```diff
@@ -165,6 +177,14 @@ int gssntlm_get_lm_compatibility_level(void);
 void gssntlm_int_release_name(struct gssntlm_name *name);
 void gssntlm_int_release_cred(struct gssntlm_cred *cred);
 
+size_t gssntlm_get_attrs_count(const struct gssntlm_name_attribute *attrs);
+int gssntlm_copy_attrs(const struct gssntlm_name_attribute *src,
+                       struct gssntlm_name_attribute **dst);
+struct gssntlm_name_attribute *
+    gssntlm_find_attr(struct gssntlm_name_attribute *attrs,
```

Comment:
Spurious indentation.

Reply:
Please propose the better solution, keeping '80 characters' limit in mind.

Reply:
See above

Reply:
yep, fixing it according to example you gave...